kobo/apps/openrosa/apps/logger/models/attachment.py

```diff
@@ -43,7 +42,12 @@ def get_queryset(self):
 
 
 class Attachment(models.Model, AudioTranscodingMixin):
-    uid = KpiUidField(uid_prefix='at')
+    uid = models.CharField(
+        max_length=23,
+        null=True,
+        blank=True,
+        db_index=True,
+    )
```

Comment:
Let's add a comment, this is going to be replaced with `KpiUidField` in a future release.

Moreover, we need to mimic `KpiUidField` behaviour, i.e.: `uid` needs to be to populated with `KpiUidField.generate_unique_id('att')` for new attachments.  Once again, this should be temporary until `KpiUidField` is in place.